From e92da4c120fefeafab883dc3f51d5f0e314ea784 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Thu, 23 Nov 2017 21:48:47 +0800
Subject: [PATCH 2925/5242] MLK-16820-3 arm64: dts: fsl-imx8qxp-mek: use
 sink-disable for typec

commit  f057029851a8be33bc5ae03337cf731794b929db from
https://source.codeaurora.org/external/imx/linux-imx.git

As we need drp config for typec data role, so change the port type
to be drp and add a sink-disable property for it.

Signed-off-by: Li Jun <jun.li@nxp.com>
Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 9070c29..701859c 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -654,7 +654,8 @@
 		max-snk-mv = <9000>;
 		max-snk-ma = <1000>;
 		op-snk-mw = <9000>;
-		port-type = "dfp";
+		port-type = "drp";
+		sink-disable;
 		default-role = "source";
 		status = "okay";
 	};
-- 
1.7.9.5

